### ISSUE TITLE
Getting error 500 when calling /user/login

### ISSUE THREAD

**Issue by phorward** [viur-framework/server#48]
There are two things relating to this issue:

1. Can there be a default-login method presented or some kind of "select a way to login", in case there are multiple logins, or a direct redirection to the valid login method in case there is only one?
2. The error points to a "fixme" - can you resolve this?

![screenshot-2018-3-6 500 - internal server error](https://user-images.githubusercontent.com/16870072/37031364-c7e45f5a-213e-11e8-8da1-70d81e04a7a0.png)

To reproduce, just create  a new, empty project from [base](https://github.com/viur-framework/base), then call http://localhost:8080/user/login.

**Comment by phorward** [comment]
Hi there! This problem still exists and is quite uncool. Can we find a solution for it in 2.4?

**Comment by tsteinruecken** [comment]
Would also defer this to 3.0 where the whole user module must be rewritten, i don't see an easy fix for it now...

**Comment by phorward** [comment]
Wouldn't it be a fix to show a html page where all login methods are linked? just in the default case something like 

```jinja2
<ul>
{% for method in authMethods %}
   <li><a href="/user/auth_{{method}}/login">{{method}}</a></li>
{% endfor %}
</ul>
```

**Comment by phorward** [comment]
#159 serves as a draft for a solution on this issue.

**Comment by tsteinruecken** [comment]
Looks you're right. I thought vi and admin would depend on this function to determine which authentication methods are supported. Checked both and it turns out they run on a hack (getAuthMethods) instead. For for 2.x okay for me.

**Comment by phorward** [comment]
fixed with #159
